### ISSUE TITLE
FamixTFile #entities slot should hold a FMMany.

### ISSUE THREAD

**Issue by juliendelplanque** [moosetechnology/Moose#1431]
Currently it hold a `FMOne`:
```
Trait named: #FamixTFile
	 uses: {}
	 slots: { #entities => FMOne type: #FamixTWithFiles opposite: #containerFiles. 
				#name }
	 category: 'Famix-Traits-File'
```

It should be:
```
Trait named: #FamixTFile
	 uses: {}
	 slots: { #entities => FMMany type: #FamixTWithFiles opposite: #containerFiles. 
				#name }
	 category: 'Famix-Traits-File'
```

If I am right.

What do you think @pavel-krivanek ?

**Comment by juliendelplanque** [comment]
After a discussion with @pavel-krivanek it is a bug, I will fix it.

**Comment by badetitou** [comment]
fixed #1434
